src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -118,6 +119,7 @@ export class InstructorCoursesPageComponent implements OnInit {
   ngOnInit(): void {
     this.route.queryParams.subscribe((queryParams: any) => {
       this.user = queryParams.user;
+      this.isAddNewCourseFormExpanded = queryParams.addNewCourse;
```

Comment:
Mind the boolean naming convention `addNewCourse`

Reply:
Updated